libs/utils/decorate.js

```diff
@@ -77,20 +77,20 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
 }
 
 export function decorateBlockBg(block, node) {
-  node.classList.add('background');
-  if (node.childElementCount > 1) {
-    const viewports = ['mobile-only', 'tablet-only', 'desktop-only'];
-    if (node.childElementCount === 2) {
-      node.children[0].classList.add(viewports[0], viewports[1]);
-      node.children[1].classList.add(viewports[2]);
-    } else {
-      [...node.children].forEach((e, i) => {
-        /* c8 ignore next */
-        e.classList.add(viewports[i]);
-      });
-    }
-  }
-  if (!node.querySelector(':scope img')) {
+  const childCount = node.childElementCount;
+  if (node.querySelector('img, video, a[href*=".mp4"]') || childCount > 1) {
+    node.classList.add('background');
+    const binaryVP = [['mobile-only', 'tablet-only'], ['desktop-only']];
```

Comment:
To me this seems overly complicated. Maybe I don't understand the goal here? 

The behavior should be:

1. If only 1 breakpoint is authored (default) then mobile, tablet, desktop get that background.
2. If 2 breakpoints are authored (mobile, tablet) Mobile only shows on mobile, tablet displays and desktop inherits tablet.
3. If 3 breakpoints are authored each breakpoint gets is own background

Something is not working correctly in the second scenario.

Ideally the only breakpoints that are added are the ones that are defined/authored and the mobile first cascade of css does its thing. 

[Example page ](https://ebartholomew-aside-bg-breakpoints--milo--adobecom.hlx.page/drafts/rclayton/test/background-per-breakpoint)

In the first aside on the page, i have 2 breakpoints defined. Mobile (Red), Tablet (image). I would expect the image would display on tablet and desktop. But, what happens is the image only shows on desktop and the Red color is displayed on mobile and tablet.

Reply:
@rgclayton this is just a refactoring of the same behavior this function had before -- both versions do the same thing. If you look at the Marquee and its proprietary `decorateBlockBg()`, it also combines mobile and tablet into the same effective "breakpoint", with desktop being separate (not inheriting from tablet).

I'm happy to change it up though, if `(mobile, (tablet, desktop))` is indeed the new direction we want to go in?

<img width="502" alt="Screenshot 2023-09-06 at 12 07 07 PM" src="https://github.com/adobecom/milo/assets/79870969/90374461-a9fc-4afc-a2ea-9dcf8b9900e7">

Reply:
@rgclayton Here's an example page showing current behavior on the `main` branch

https://main--milo--adobecom.hlx.page/drafts/ebartholomew/two-breakpoints

Reply:
Ahh. Got it. I don't think this needs fixed in this ticket then if it was already existing. To me it seems like a bug with how the viewports are set up.

@ryanmparrish can you provide any clarity here?

Reply:
@rgclayton - You're right, this is set up wrong IMO for the case w/ just 2 bg cells. 
It should be
`[mobile][tablet/desktop]`
It currently is. 
`[mobile/tablet][desktop]`

I'm open as to wether we fix this here or in a separate PR. But I do think we should address it and come correct.. (that probably means looking at potential usage in the wild if thats even searchable?) 
Otherwise we leave it as is and deal w/ the weird anti-patern in this small edge case. I looked at my, [original draft](https://main--milo--adobecom.hlx.page/drafts/rparrish/marquee) and see it wasn't quite right. ex. '2 viewport images (mobile/tablet, desktop)' :(

Reply:
>It should be
[mobile][tablet/desktop]
It currently is.
[mobile/tablet][desktop]

Yep. 

@elan-tbx if you have the bandwidth I think we fix it in this PR and squash the issue. Otherwise code is just getting added on to it that tangles it more. Sooner it can get fixed the better.  

cc: @ryanmparrish

Reply:
@rgclayton yep, I have the bandwidth. Just to make sure -- since the Marquee has its own function that also does this, should I also make the change in that block in this PR? Feels kind of out of scope of this PR's original purpose, but might be worth making an exception for this.

cc: @ryanmparrish